en/peripherals/frsky_telemetry.md

```diff
@@ -198,6 +203,20 @@ You will need to connect via a UART to S.PORT adapter board, or a [ready-made ca
 <!-- ideally add diagram here -->
 
 
+
+## Firmware Setup {#firmware}

```

Comment:
@bkueng FYI you will note that in a number of places I have this section duplicated. I would love to have this information generically, but AFAIK there is no generic way to add all rangefinder drivers, just the serial ones.

Do you think we might be better off being specific and replacing this with a specific note about just uncommenting the driver for the FMUv2 target (everywhere)?

Reply:
Hmm, not sure what's best. One one hand we might have more flash-limited boards/targets in future that we want to support, and on the other hand we have the Bootloader update for FMUv2, so that most people are not affected anymore.
So maybe a minimal note here with the driver that needs to be added (`drivers/telemetry/frsky_telemetry`) and a link to a more extensive & general page would do it?

Reply:
OK, I've fixed this with a troubleshooting section at the end of the the *Serial Port Configuration* page. The links to this are not very thin in the peripheral pages.